packages/twenty-front/src/modules/action-menu/components/RightDrawerActionMenuDropdown.tsx

```diff
@@ -59,7 +59,9 @@ export const RightDrawerActionMenuDropdown = () => {
           RightDrawerActionMenuDropdownHotkeyScope.RightDrawerActionMenuDropdown,
       }}
       data-select-disable
-      clickableComponent={<Button title="Actions" shortcut="⌘O" />}
+      clickableComponent={
+        <Button title="Actions" hotkeys={[`${getOsControlSymbol()}`, 'O']} />
```

Comment:
I think there is no need to wrap the function call in a template literal as the `getOsControlSymbol` function always returns a string. Am I missing something?

```suggestion
        <Button title="Actions" hotkeys={[getOsControlSymbol(), 'O']} />
```